src/controllers/DoctorController.java

```diff
@@ -59,7 +63,9 @@ private Command LoadPatient() {
             String patientUsername = doctorScreenView.loadPatientPrompt();
             PatientData loadedPatientData = patientManager.getUserData(patientUsername);
             if (loadedPatientData != null){
-                doctorScreenView.showSuccessLoadingPatient(new ContactManager(getDatabase()).getContactData(loadedPatientData));
+                logManager.addLog(doctorData, "loaded patient: " + patientUsername);
+                doctorScreenView.showSuccessLoadingPatient(
+                        new ContactManager(getDatabase()).getContactData(loadedPatientData));
```

Comment:
Declare this contant manager above instead of instatitating it here bit more readable